src/test/scala/fable/ConsumerSpec.scala

```diff
@@ -231,5 +235,39 @@ class ConsumerSpec extends AsyncFunSuite {
       producer.close
     }
 
+  private def createMockKafkaObjects(topic: Topic, offset: Long, topicPartition: TopicPartition) = {
```

Comment:
Since this creates a single object, does `createMockConsumer` make sense as a name?

Can we add a return type to this definition?